models/classification/ote_efficientnet_b0_multilabel_al.yaml

```diff
@@ -2,11 +2,9 @@ _base_: ./ote_efficientnet_b0_multilabel.yaml
 
 model:
     head:
-        type: CustomMultiLabelNonLinearClsHead
+        type: CustomMultiLabelLinearClsHead
```

Comment:
Is it okay @kprokofi @sovrasov?

Reply:
Yep, the head should be linear for EfficientNets as it is in [develop](https://github.com/openvinotoolkit/model_preparation_algorithm/blob/develop/models/classification/ote_efficientnet_b0_multilabel_al.yaml) branch now. I have no idea why it was changes in otx, but this PR fixes the issue.

Reply:
I don't know why this change is here, maybe the push from [8c4c328](https://github.com/openvinotoolkit/model_preparation_algorithm/commit/8c4c32840016c254e5e23addac0063633559e1d7)?

Reply:
Anyway, this PR fixed multi-label bugs :) LGTM